docs/sw-build.md

```diff
@@ -31,23 +31,25 @@ into your build script.
 
 Add the following to your `gulpfile.js`:
 
-    const swBuild = require('sw-build');
-
-    swBuild.generateSW({
-      rootDirectory: './build/',
-      dest: './build/sw.js',
-      globPatterns: ['**\/*.{html,js,css}'],
-      globIgnores: ['admin.html'],
-      templatedUrls: {
-        '/shell': ['shell.hbs', 'main.css', 'shell.css'],
-      },
+    gulp.task('build', (callback) => {
+      const swBuild = require('sw-build');
+
+      swBuild.generateSW({
```

Comment:
You need to `return swBuild.generateSW({...` here, so that the promise chain can be used to signify that the `gulp` task has completed.